example/android/app/src/main/AndroidManifest.xml

```diff
@@ -1,7 +1,12 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android">
 
     <uses-permission android:name="android.permission.INTERNET" />
+    <uses-permission android:name="android.permission.CAMERA" />
+    <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE" />
+    <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" />
```

Comment:
We will need `<uses-permission android:name="android.permission.READ_MEDIA_IMAGES"/>` for android 13

Reply:
yes, I added new